src/parser/shared/modules/items/Potion.js

```diff
@@ -35,6 +35,12 @@ class Potion extends Analyzer {
 
   constructor(...args) {
     super(...args);
+
+    if (!this.isAvailable) {
+      this.active = false;
```

Comment:
Can you check to see if this change works if you simply move the `active = false` into an extended constructor in `Healthstone`? Rather than creating a new function to extend. `active = false` should prevent all events from being called for the module and statistics from being displayed. But there is some other behind the scenes stuff to test with that as well.

Reply:
I tried that before but unfortunately that didn't seem to have much effect on the ability being available. I even setup a constructor as following for the Healthstone;

```js
constructor(...args) {
  super(...args);
  this.active = false;
}
```